pkgs/os-specific/linux/udisks/2-default.nix

```diff
@@ -22,6 +22,10 @@ stdenv.mkDerivation rec {
       substituteInPlace src/main.c --replace \
         "@path@" \
         "${utillinux}/bin:${mdadm}/sbin:/var/run/current-system/sw/bin:/var/run/current-system/sw/bin"
+      substituteInPlace data/80-udisks2.rules \
+        --replace "/bin/sh" "${stdenv.shell}" \
+        --replace "/sbin/mdadm" "${mdadm}/sbin/mdadm" \
+        --replace " sed " " ${gnused}/bin/sed "
```

Comment:
When doing this, `/run/current-system/sw/bin` is probably more canonical as well, but I wonder what is needed from that path...